### PR TITLE
Here is an example of OAUTH2 using GitHub

### PR DESCRIPTION
**Pull request by SoftExpert**
Created this sample app by adapting the code from https://github.com/Skarlso/google-oauth-go-sample.
I hope it is clear and secure enough.

### CODE REVIEW

File: testOauth2/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (koddr) [inline testOauth2/.vscode/launch.json:1]: Please add all files from `.vscode` folder to `.gitignore`. It's your personal VSCode settings.

File: testOauth2/go.mod [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+go 1.16
+
+require (
+	github.com/antigloss/go v0.0.0-20201201072909-f29271b13566
+	github.com/gofiber/fiber/v2 v2.6.0
+	github.com/gofiber/template v1.6.7
+	github.com/joho/godotenv v1.3.0
+	github.com/klauspost/compress v1.11.12 // indirect
+	github.com/segmentio/encoding v0.2.14
+	github.com/valyala/fasthttp v1.22.0 // indirect
```
Comment (koddr) [inline testOauth2/go.mod:12]: Please run `go mod tidy` to delete `indirect` dependencies.

File: testOauth2/models/models.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+type OAuthAccessResponse struct {
+	AccessToken      string `json:"access_token"`
+	TokenType        string `json:"token_type"`
+	Scope            string `json:"scope"`
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+	ErrorURI         string `json:"error_uri"`
+}
+
+// SYSLOG ...
```
Comment (koddr) [inline testOauth2/models/models.go:21]: Please add more understandable comments or make local vars or structs (without comments).

File: testOauth2/go.mod (hunk 2 above)
Comment (SoftExpert) [reply testOauth2/go.mod:13]: Impossible to get rid of indirect dependencies - they are brought up by fiber, by the json encoder and by fasthttp.
Gladly taking any suggestion ...

File: testOauth2/models/models.go (hunk 3 above)
Comment (SoftExpert) [reply testOauth2/models/models.go]: Done

File: testOauth2/.vscode/launch.json (hunk 1 above)
Comment (SoftExpert) [reply testOauth2/.vscode/launch.json:1]: Added .gitignore file which got itself added to the repository.
I must have missed something ...

**Pull request merged**